core/com.b2international.snowowl.datastore/src/com/b2international/snowowl/datastore/remotejobs/RemoteJob.java

```diff
@@ -44,21 +45,25 @@
 	private final String id;
 	private final ServiceProvider context;
 	private final Request<ServiceProvider, ?> request;
+	private final boolean autoClean;
 	
 	private String response;
 	private String user;
+	private IStatus status;
```

Comment:
This should be either `transient` field or not a field at all. I suggest moving this as a variable into the `run(IProgressMonitor monitor)` method.